File: 4st/Eunseong/피자 나눠 먹기(1).c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int solution(int n) {
    int answer = 0;
    if (n % 7 == 0) {
        answer = n / 7;
    } else {
        answer = (n / 7) + 1;
    }
    return answer;
}

File: 4st/Seohyun/programmer/피자 나눠 먹기(1).java
class Solution {
    public int solution(int n) {
        int answer;
        if(n%7==0)
            answer=n/7;
        else if(n<7)
            answer=1;
        else if(n>7 && (n%7)>0)
            answer=n/7+1;
        return answer;
    }
}